RetsTalk/RetsTalk/Chat/Controller/ChattingViewController.swift

```diff
@@ -73,3 +105,12 @@ extension ChattingViewController: UITableViewDelegate, UITableViewDataSource {
         return cell
     }
 }
+
+// MARK: constants
```

Comment:
```suggestion
// MARK: - Constants
```

외부의 MARK문은 구분선이 보이도록 하이픈을 포함하는 것이 더 좋아보입니다. 그리고 대문자 시작.